components/host.py

```diff
@@ -1158,7 +1158,7 @@ def get_next_classical(self, sender_id, wait=-1):
         wait_start_time = time.time()
         while (time.time() - wait_start_time < wait or wait==-1)\
                                                     and ret == None:
-            time.sleep(0.01)
+            self.delay(0.01)
```

Comment:
Ah, here I meant `time.sleep(self.delay)` since 0.01 is slightly a [magic number](https://en.wikipedia.org/wiki/Magic_number_(programming)).

Reply:
oops :D 
I deleted the delay completely, since the delay shouldn't be too high, it would significantly reduce the speed of some applications. I guess, we do not really need a delay here anyways.